src/storage/mod.rs

```diff
@@ -1753,4 +1786,99 @@ mod tests {
         );
         storage.stop().unwrap();
     }
+
+    #[test]
+    fn test_raw_delete_range() {
+        let read_pool = ReadPool::new("readpool", &readpool::Config::default_for_test(), || {
```

Comment:
I see too many places use the same way to create the test read pool, can we extract them to one simple function?

@breeswish

Reply:
let's do this in another PR?

Reply:
agree. let's do it in another PR.